server/package.json

```diff
@@ -104,6 +104,7 @@
   "devDependencies": {
     "@swc/core": "1.3.6",
     "@types/express": "4.17.14",
+    "@types/mocha": "10.0.1",
```

Comment:
could be removed or replaced with jest

Reply:
if removed, when deploying the tests folder, it won't work for TS. I added mocha/types since we use mocha also.